rts/Lua/LuaUnsyncedRead.cpp

```diff
@@ -602,6 +603,35 @@ int LuaUnsyncedRead::GetMiniMapDualScreen(lua_State* L)
 }
 
 
+int LuaUnsyncedRead::GetSelectionBox(lua_State* L)
+{
+	float3 bl, br, tl, tr;
+	if (!mouse->GetSelectionBoxVertices(bl, br, tl, tr)) {
+		lua_pushnil(L);
+
+		return 1;
+	}
+
+	const auto bottomLeft = camera->CalcWindowCoordinates(bl);
+	const auto topRight   = camera->CalcWindowCoordinates(tr);
+
+	lua_createtable(L, 2, 0);
+
+	lua_pushnumber(L, bottomLeft.x);
+	lua_rawseti(L, -2, 1);
+	lua_pushnumber(L, bottomLeft.y);
+	lua_rawseti(L, -2, 2);
+
+	lua_createtable(L, 2, 0);
+
+	lua_pushnumber(L, topRight.x);
+	lua_rawseti(L, -2, 1);
+	lua_pushnumber(L, topRight.y);
+	lua_rawseti(L, -2, 2);
+
+	return 2;
```

Comment:
Convert output to just 4 lua numbers, I don't think tables here make sense

Reply:
Done